apps/backoffice-v2/src/pages/Entities/components/Cases/hooks/useCases/useCases.tsx

```diff
@@ -86,11 +86,15 @@ export const useCases = () => {
   }, []);
 
   useDocumentListener('keydown', event => {
-    if (!event.ctrlKey) return;
+    if (!event.ctrlKey || !event.shiftKey) return;
```

Comment:
Prior to these changes it was not possible to use shortcuts like `CTRL + A`, `CTRL +C`, etc.